wrapcheck/wrapcheck.go

```diff
@@ -91,7 +93,24 @@ func NewAnalyzer(cfg WrapcheckConfig) *analysis.Analyzer {
 }
 
 func run(cfg WrapcheckConfig) func(*analysis.Pass) (interface{}, error) {
+	var Err error
+
+	for i := range cfg.IgnoreSigRegexps {
+		re, err := regexp.Compile(cfg.IgnoreSigRegexps[i])
+		if err != nil {
+			Err = fmt.Errorf("unable to parse regexp: %v at %s\n", err, cfg.IgnoreSigRegexps[i])
+			log.Printf(Err.Error())
+			break
+		}
+		cfg.ignoreSigRegexpCompiled = append(cfg.ignoreSigRegexpCompiled, re)
```

Comment:
I don't think this value belongs on the config struct. I'd suggest just defining a local variable to be used.

Reply:
make sense.

Reply:
I made a fix, however, I am not really happy passing the additional argument to `reportUnwrapped`.  IMO, `ignoreSigRegexpCompiled` can be a part of configuration (even if its not public part of it,)